### ISSUE TITLE
Installing when grumphp.yml is not created or is empty causes warnings

### ISSUE THREAD

**Issue by aderuwe** [phpro/grumphp#2]
```
Writing lock file
Generating autoload files

Warning: array_key_exists() expects parameter 2 to be array, null given in vendor/phpro/grumphp/src/GrumPHP/Configuration/TaskCompilerPass.php on line 27
```

**Comment by aderuwe** [comment]
It also means that the bin dir might be set to the default incorrectly.

**Comment by veewee** [comment]
The warning is fixed in master. The paths are still an issue.

**Comment by veewee** [comment]
First part of the issue is fixed, the other part will be solved by #4.
